fastlane/Matchfile

```diff
@@ -1,6 +1,10 @@
 # This Matchfile has the shared properties used for all signing types
 
 # Store certs/profiles encrypted in Google Cloud
-storage_mode("google_cloud")
-google_cloud_bucket_name("a8c-fastlane-match")
-google_cloud_keys_file(".configure-files/google_cloud_keys.json")
+storage_mode('google_cloud')
+google_cloud_bucket_name('a8c-fastlane-match')
+# Interestingly, Using the path with '~/.configure/...' results in Fastlane
+# failing to find the file.
+SECRETS_ROOT = File.join(Dir.home, '.configure/simplenote-ios/secrets')
```

Comment:
I experienced that with the Google Cloud Key value. Sounds like a bug in Fastlane that we could help address. I haven't had the chance to dig into it, yet.